src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#logo" class="logo"><img src="images/logo.png" alt="logotip"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active"><a href="#apple" class="link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="link">Samsung</a></li>
```

Comment:
please write children from new line and add spaces to them. don't forget to add empty line between siblings.
![image](https://user-images.githubusercontent.com/22824431/222443942-9b581cf1-09ef-49bd-8b4d-2b5eb30e4586.png)

```suggestion
          <li class="nav__item is-active">
            <a href="#apple" class="link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="link">
              Samsung
            </a>
          </li>
```